### PR TITLE
fix: publish types

### PR DESCRIPTION
**Pull request by ivanstanev**
It would be useful of the users of this library to know the exported types, so make sure they are published in the package.
This is a part of a wider effort to add more type information to the plugin.

- [x] Ready for review
- [x] Follows CONTRIBUTING rules
- [ ] Reviewed by Snyk internal team

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +18,8 @@
     "format": "prettier --loglevel warn --write '**/*.ts' && tslint --fix --format stylish '**/*.ts' --exclude 'node_modules/**'",
     "test": "npm run lint && npm run unit-test",
     "unit-test": "tap test/system test/lib -R=spec --timeout=300",
-    "prepare": "npm run build"
+    "prepare": "npm run build",
+    "prepublish": "tsc"
```
Comment (FauxFaux) [inline package.json]: I think it runs `prepare` during `prepublish` anyway?

Comment (ivanstanev) [reply package.json]: Correct, removed it!

File: tsconfig.json [hunk 2]
```diff
@@ -58,5 +58,9 @@
   },
   "include": [
     "./lib/**/*"
+  ],
+  "exclude": [
```
Comment (FauxFaux) [inline tsconfig.json]: I.. think you have to ship `dist`.

This really isn't my area of speciality. Please consult a node expert.

**Comment by ivanstanev** [comment]
I don't understand why everything works well locally but fails on CI...

**Comment by ivanstanev** [comment]
Giving up on this one, spent a lot of time on weird errors

**Pull request closed without merging**